lib/board.rb

```diff
@@ -73,6 +79,26 @@ def renders(player, arg = false)
 "B #{@cells["B1"].render(arg)} #{@cells["B2"].render(arg)} #{@cells["B3"].render(arg)} #{@cells["B4"].render(arg)} \n" +
 "C #{@cells["C1"].render(arg)} #{@cells["C2"].render(arg)} #{@cells["C3"].render(arg)} #{@cells["C4"].render(arg)} \n" +
 "D #{@cells["D1"].render(arg)} #{@cells["D2"].render(arg)} #{@cells["D3"].render(arg)} #{@cells["D4"].render(arg)} \n"
+
+
+    # width_num = (1..@size).to_a
+    # height_num = width_num.map(&:clone)
+    # width = width_num.map {|num| num.to_s}
+    # height = height_num.map {|num| (num += 64).chr}
+    #
+    # print "  "
+    # width.each do |num|
+    #   print "#{num} "
+    # end
+    # print "\n"
+    #
+    # split_coordinate = @cells.keys.map {|coordinate| coordinate.split''}
+    #
+    #
+    # height.each do |letter|
+    #   print letter
+    #   while letter ==
```

Comment:
I'm assuming that this is still being adjusted. Otherwise, what are you expecting this to produce?